scripts/zones/AlTaieu/mobs/Jailer_of_Prudence.lua

```diff
@@ -19,6 +24,17 @@ end;
 -----------------------------------
 
 function onMobSpawn(mob)
+    mob:AnimationSub(0); -- Mouth closed
+    mob:addStatusEffect(EFFECT_FLEE,100,0,30);
```

Comment:
Wasn't there a way to make it so when this status wears there wouldn't be a message?